.github/workflows/ci.yml

```diff
@@ -19,7 +19,6 @@ on:
         default: "false"
 
 env:
-  GO_VERSION: "1.18.5" # for non sandboxed e2e tests
```

Comment:
Note that the release pipeline is still on 1.18; does the CI matrix also test against that version?

Reply:
The release pipeline in `docker/docker-ce-packaging`, right? (IIRC we settled on it calls `make build` from the `Makefile` in this repo, which does a "local" `go build` inside of its container but not using `buildx bake` for the actual Compose build)

How do we generally handle that for other projects packaged by it? Something like this?
* Upgrade project CI to default to Go `n+1`
* Continue running project CI against Go `n` & `n+1`
* Once all projects on Go `n+1`, change packaging to use Go `n+1`
* Drop Go `n` from project CI

cc @glours if you have opinions